docs/app/Examples/addons/TextArea/Types/TextAreaExample.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react'
+import { Form, TextArea } from 'semantic-ui-react'
+
+class TextAreaExample extends Component {
```

Comment:
We prefer to use a stateless functional component when possible, could you update this to use a stateless functional component?  I think we should also show the simplest default `<Textarea />` usage here.  The current usage is really a feature of the `<Form />`.

On a side note, `<Form.TextArea />` is a `<TextArea />` wrapped in a `<Form.Field />` so the wrapping form field here is creating a duplicate nested form field.

```jsx
const TextAreaExample = () => (
  <TextArea placeholder='TextArea placeholder text' rows='3' />
)
```